### ISSUE TITLE
Tasks no longer showing up in label-studio projects

### ISSUE THREAD

**Issue by WillieMaddox** [heartexlabs/label-studio#6251]
Today I updated my OS with the standard `apt update; apt upgrade`.  After restarting the machine and logging in to label-studio, the main page where all my projects are listed displayed good as usual. But whenever I click on a project, none of the tasks show up.

I've tried chrome, edge and brave browsers, all have the same issue.

**Screenshots**

Before when I would click on a project, the little aardvark would appear in the middle of the screen while the projects was loading.  Now it appears in the upper left hand corner of the screen:

![image](https://github.com/user-attachments/assets/9949d7a2-78ee-4e0d-8ce5-5b98c9f19366)

This is what I see after a project loads:

![image](https://github.com/user-attachments/assets/e156a9fe-f609-4990-94ef-e7cb3a9a825d)

If I click on the blue filters arrow, everything goes away except for the topmost header:

![image](https://github.com/user-attachments/assets/135a1c53-359a-4cb8-9c19-908e6a63425a)

This is what I see just after I click on one of the other views:

![image](https://github.com/user-attachments/assets/9a752ef1-cf94-410f-a42d-53e6e13eef59)

**Environment:**
 - OS: Ubuntu 24.04
 - Label Studio Version: 1.13.0

**Additional context:**
I run label-studio from the command line: `docker compose -f docker-compose.yml up`.  This is the way I've always done it, so nothing new here.

I also noticed some `UserWarning`'s in the logs:
```sh
app-1    | /usr/local/lib/python3.10/dist-packages/pydantic/_internal/_fields.py:160: UserWarning: Field "model_provider_connection_create" has conflict with protected namespace "model_".
app-1    | 
app-1    | You may be able to resolve this warning by setting `model_config['protected_namespaces'] = ()`.
app-1    |   warnings.warn(
app-1    | 
app-1    | /usr/local/lib/python3.10/dist-packages/pydantic/_internal/_fields.py:160: UserWarning: Field "model_provider_connection_view" has conflict with protected namespace "model_".
app-1    | 
app-1    | You may be able to resolve this warning by setting `model_config['protected_namespaces'] = ()`.
app-1    |   warnings.warn(
app-1    | 
app-1    | /usr/local/lib/python3.10/dist-packages/pydantic/_internal/_fields.py:160: UserWarning: Field "model_provider_connection_change" has conflict with protected namespace "model_".
app-1    | 
app-1    | You may be able to resolve this warning by setting `model_config['protected_namespaces'] = ()`.
app-1    |   warnings.warn(
app-1    | 
app-1    | /usr/local/lib/python3.10/dist-packages/pydantic/_internal/_fields.py:160: UserWarning: Field "model_provider_connection_delete" has conflict with protected namespace "model_".
app-1    | 
app-1    | You may be able to resolve this warning by setting `model_config['protected_namespaces'] = ()`.
app-1    |   warnings.warn(
app-1    | 
```

**Comment by TRYOKETHEPEN** [comment]
I got the same issue. 
My docker container is running on Ubuntu20.04.6, and I interview the webUI on Win10 . 
It's **OK** with Chrome 127.0.6533.120 and Edge 127.0.2651.105, **NG** with Chrome 128.0.6613.85 and Edge 128.0.2739.42 .
I've tried label-studio version: 1.12.1, 1.13.0 .

**Comment by CsterKuroi** [comment]
same issue with chrome 128.0.6613.85

**Comment by neocwl** [comment]
same issue

**Comment by punnerud** [comment]
Seems like its after Chrome update. I had a team mate with the same problem, and switching browser fixed it.
I worked for me (before updating Chrome), and stoped after update.

Tried to clean cache and login one more time, without any difference.
So its clearly linked to the new version of Chrome.

**Comment by makseq** [comment]
We are on it. It would be very helpful if you could show a screenshot with the browser console open and the errors in it.

**Comment by punnerud** [comment]
> We are on it. It would be very helpful if you could show a screenshot with the browser console open and the errors in it.

<img width="1501" alt="ls_chrome_bug" src="https://github.com/user-attachments/assets/425dcf71-b1e8-4342-adc2-f00edaad5ea5">

**Comment by CsterKuroi** [comment]
![image](https://github.com/user-attachments/assets/73bbe2d3-0d5f-49b5-8d6a-2e131f955466)

**Comment by dev6699** [comment]
Having the same issue when using google chrome 128 and labelstudio version: `1.13.0 and below`, but able to resolve with labelstudio version `1.13.1`
![lbl-1 13 1](https://github.com/user-attachments/assets/9a661d53-b278-46a7-8cb6-a497dad7769e)
![lbl-1 13 0](https://github.com/user-attachments/assets/54d8376a-e379-408a-92c2-6ae8952579fb)

**Comment by makseq** [comment]
Ok, I've checked both versions: 1.13.0 and 1.13.1. 
1.13.0 has this issue with the latest browsers. 
1.13.1 works well. 

The solution is to upgrade to 1.13.1.

**Comment by matt3o** [comment]
@makseq  Thanks for the quick fix, we ran into this as well! Are there any plans to update the Docker :latest image with the fixes soon?

**Comment by WillieMaddox** [comment]
@makseq Upgrading to 1.13.1 worked for me too.

**Comment by makseq** [comment]
@matt3o docker should be fixed now. Latest tag pointed to 1.13.0, however, 1.13.1 has been already released.

**Comment by JKYang01** [comment]
Hi labelstudio team,
I have the same problem
I deployed labelstudio on AWS EKS cluster through kubernets and I was following the documents in: https://github.com/HumanSignal/charts/tree/master/heartex/label-studio
labelstudio version 1.13.1


I encountered two strange behaviors of label studio. it worked fine when I deploy it for the first week, so I am very confused, what happened there.

1. label studio cannot show the tasks on google chrome.
It works fine when I change the browser to Firefox.

2. When I create projects and then click to “labeling all tasks” it only show one task. even though there are 28 tasks shows up on the page. I can only access the tasks by clicking the tasks directly, if the tasks doesn’t show up, (like on google chrome) I cannot do anything.


My label studio set up:

labelstudio version: 1.13.1

I deployed labelstudio on AWS EKS, I installed and updated labelstudio using helm, and the following global envrionment settings of my label studio: 

helm upgrade labelstudio heartex/label-studio -namespace label-studio-dev \
–set global.extraEnvironmentVars.LABEL_STUDIO_DISABLE_SIGNUP_WITHOUT_LINK=“true”
–set [global.extraEnvironmentVars.LABEL_STUDIO_USERNAME="xxx@enfej.com 
--set global.extraEnvironmentVars.LABEL_STUDIO_USERNAME=%22xxx@enfej.com"
–set global.extraEnvironmentVars.LABEL_STUDIO_PASSWORD=“mypassword”
–set global.extraEnvironmentVars.SSRF_PROTECTION_ENABLED=“true”
–set global.extraEnvironmentVars.LABEL_STUDIO_HOST=“http://xxxx.xxxx.com/”
